dashboard/test/mailers/previews/pd_teacher_application_mailer_preview.rb

```diff
@@ -5,6 +5,8 @@ class PdTeacherApplicationMailerPreview < ActionMailer::Preview
 
   %w(
     confirmation
+    complete_application_initial_reminder
+    complete_application_final_reminder
```

Comment:
nit: should these be `incomplete_` instead of `complete_`? It confused me at first given that everything else uses "incomplete" but I'm not familiar with naming conventions here

Reply:
I'm confused by this too. The email spec https://docs.google.com/document/d/1I2WXc7REV0QdZ5EaxjHBj36CDkawmcw-xQyEvZcFHao/edit#heading=h.3npnbftqigdv calls these emails "Complete your application." But it's for incomplete applications that these get sent –– for incomplete applications, we email them to "complete your application."

I could change all the service instances to `complete_reminder` or `complete_application_reminder` or `reminder_to_complete`. It's helpful for me to know that these only get sent on incomplete apps, but maybe that's not outwardly useful? Thoughts?

Reply:
I would prefer if they were consistent but I don't have a strong opinion as to which names are used. If the naming is normally consistent with the subject of the email then maybe `complete_` is better? Otherwise, I'd lean in the same direction as you and use `incomplete_`

Reply:
Good call –– the `registration_reminder` matches the name of the mailer, so I changed it to `complete_application_reminder`